kedro-org/kedro-viz
# Temporary fix for graphNew error in local development

## Description

Partially resolves [#1913](https://github.com/kedro-org/kedro-viz/issues/1913)

Fixes the below error with a temporary work-around:
```
graph.js:36 Uncaught (in promise) TypeError: instance.graphNew is not a function
    at layout (graph.js:36:1)
    at worker.js:58:1
    at graph.js:53:1
    at Object.dispatch (index.js:16:1)
    at Object.dispatch (<anonymous>:6:7384)
    at store.js:19:1
    at index.js:17:1
    at Object.dispatch (redux.js:297:1)
    at s (<anonymous>:3:5093)
    at index.js:20:1
```    

## Development notes

There are some compatibility issues with workerize-loader and Webpack v5 which is affecting the local dev env. As @jitu5 and I discussed, this issue happened mostly after node18 upgrade. As Jitendra pointed out to these [commits](https://github.com/kedro-org/kedro-viz/pull/1848/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L132)

Also a similar issue - https://github.com/developit/workerize-loader/issues/77 , for which some are suggesting having a simple own library. Since this issue happens in dev env, I felt we opt for a less effort work-around. Further looking at the fix, I realized this might be an issue with cache.

**What does this PR do -**

Introduces a script - `"start:dev": "rm -rf node_modules/.cache && npm start",` at package.json which can be used in local development instead of doing `npm start`. Now developers can make use of this script by running `npm run start:dev` which helps removing the cache and resolving the error.

## QA notes

- `npm run start:dev` in local should not throw the above error
- All CI tests should pass

## Checklist
